src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public enum Colors {
+    violet,
+    purple,
```

Comment:
1. class/enum names should be in singluar form
2. enum members are constant so should have proper NAMING_CASE
```suggestion
public enum Color {
    VIOLET,
    PURPLE,
```